sdk/communication/Azure.Communication.PhoneNumbers/README.md

```diff
@@ -42,28 +42,28 @@ All geographic phone plans within the same country are grouped into a phone plan
 
 Phone Number clients can be authenticated using connection string acquired from an Azure Communication Resources in the [Azure Portal][azure_portal].
 
-```C# Snippet:CreatePhoneNumberAdministrationClient
+```C# Snippet:CreatePhoneNumbersClient
 // Get a connection string to our Azure Communication resource.
 var connectionString = "<connection_string>";
-var client = new PhoneNumberAdministrationClient(connectionString);
+var client = new PhoneNumbersClient(connectionString);
 ```
 
 Phone Number clients also have the option to authenticate with Azure Active Directory Authentication. With this option,
 `AZURE_CLIENT_SECRET`, `AZURE_CLIENT_ID` and `AZURE_TENANT_ID` environment variables need to be set up for authentication.
 
-```C# Snippet:CreatePhoneNumberWithTokenCredential
-var endpoint = "<endpoint_url>";
+```C# Snippet:CreatePhoneNumbersClientWithTokenCredential
+// Get an endpoint to our Azure Communication resource.
+var endpoint = new Uri("<endpoint_url>");
 TokenCredential tokenCredential = new DefaultAzureCredential();
-var client = new PhoneNumberAdministrationClient(new Uri(endpoint), tokenCredential);
+client = new PhoneNumbersClient(endpoint, tokenCredential);
 ```
 
-### Reserving and acquiring numbers
+### Phone Number Types overview
+Phone numbers come in two types: Geographic and Toll-Free. Geographic phone plans are phone plans associated with a location, whose phone numbers' area codes are associated with the area code of a geographic location. Toll-Free phone plans are phone plans not associated location. For example, in the US, toll-free numbers can come with area codes such as 800 or 888.
 
-Phone numbers reservation can be performed through the reservation creation API by providing a phone plan id, an area code and quantity of phone numbers. The provided quantity of phone numbers will be reserved for ten minutes. This reservation of phone numbers can either be cancelled or purchased. If the reservation is cancelled, then the phone numbers will become available to others. If the reservation is purchased, then the phone numbers are acquired for the Azure resources.
+### Searching and Purchasing and Releasing numbers
```

Comment:
Searching, Purchasing and Releasing Phone Numbers

Reply:
Done